vmm/Cargo.toml

```diff
@@ -13,6 +13,7 @@ kvm-ioctls = { git = "https://github.com/rust-vmm/kvm-ioctls", branch = "master"
 libc = ">=0.2.39"
 log = "*"
 net_util = { path = "../net_util" }
+nix = "0.14.1"
```

Comment:
We need to think carefully about adding a dependency.

Reply:
Yes, add it to use unistd::mkstemp. I see nix has maintained actively in https://github.com/nix-rust/nix, 1686 commits, 182 contributors, and a start number of 967. it seems to be safe for reference. Any other concern?

Reply:
Pulling in a big dependency for two functions and one constant seems bad.

Reply:
Here is the implementation of nix::mkstemp. I think you can use libc crate directly.

``
pub fn mkstemp<P: ?Sized + NixPath>(template: &P) -> Result<(RawFd, PathBuf)> {
    let mut path = try!(template.with_nix_path(|path| {path.to_bytes_with_nul().to_owned()}));
    let p = path.as_mut_ptr() as *mut _;
    let fd = unsafe { libc::mkstemp(p) };
    let last = path.pop(); // drop the trailing nul
    debug_assert!(last == Some(b'\0'));
    let pathname = OsString::from_vec(path);
    try!(Errno::result(fd));
    Ok((fd, PathBuf::from(pathname)))
}`


```

Reply:
Yes, since this is a simple one, it could be replaced by calling libc directly. The change is done.